### ISSUE TITLE
Bundler not finding Contour 1.1.2.pre Prerelease Gem on bundle update

### ISSUE THREAD

**Issue by remomueller** [bundler/bundler#2209]
[gist.github.com/4250968](https://gist.github.com/4250968)

I'm the owner of the Contour gem, and pushed a [new prerelease version 1.1.2.pre](https://rubygems.org/gems/contour/versions/1.1.2.pre) to RubyGems.  I've done [prerelease versions in the past](https://rubygems.org/gems/contour/versions), and have always been able to specify the version then in the Gemfile, for example: <code>gem 'contour', '~> 1.1.2.pre'</code>. Then followed by a <code>bundle update</code>, the gem would install correctly.

For some unknown reason, bundler is unable to find this gem, although it is available on RubyGems: [rubygems.org/gems/contour/versions/1.1.2.pre](https://rubygems.org/gems/contour/versions/1.1.2.pre)

I've also opened an [issue on RubyGems Help](http://help.rubygems.org/discussions/problems/1462-contour-112pre-not-showing-up-on-bundle-update) in case it is actually a RubyGems error.

TL;DR Pushed a prerelease gem and Bundler has no knowledge of it.
